src/main/java/org/broadinstitute/consent/http/enumeration/DarCollectionStatus.java

```diff
@@ -1,7 +1,7 @@
 package org.broadinstitute.consent.http.enumeration;
 
 public enum DarCollectionStatus {
-  UNREVIEWED("Unreviewed"),
+  SUBMITTED("Submitted"),
```

Comment:
This seems fine, but we could also have the `value` diverge from the enum name. if the value is the UI version of this state.